src/Plugins/Loader/ManagedLoadContext.cs

```diff
@@ -40,11 +41,12 @@ internal class ManagedLoadContext : AssemblyLoadContext
             IReadOnlyDictionary<string, ManagedLibrary> managedAssemblies,
             IReadOnlyDictionary<string, NativeLibrary> nativeLibraries,
             IReadOnlyCollection<string> privateAssemblies,
-            IReadOnlyCollection<string> defaultAssemblies,
+            ICollection<string> defaultAssemblies,
```

Comment:
Can we find a way to code this without changing this? The intention behind read-only behaviors was to minimize the chance for unintended side-effects between contexts.

Reply:
Are you referring to the parameter specifically or the class field?
I suppose one option is to have read only parameter but copy to our own private collection in the constructor.

This way the API would still signal to the user that the collection is immutable/cannot be changed once instantiated.